include/pf-applications/lac/dynamic_block_vector.h

```diff
@@ -400,13 +387,11 @@ namespace dealii
         void
         swap(DynamicBlockVector &V)
         {
-          std::swap(this->block_counter, V.block_counter);
           std::swap(this->blocks, V.blocks);
           std::swap(this->size_, V.size_);
         }
 
       private:
-        unsigned int                            block_counter;
```

Comment:
I introduced `block_counter`, since I thought the system will be much more dynamic and we have to change the block size more frequently. But this is not the case...